data/sql/updates/pending_db_world/rev_1701079229296730800.sql

```diff
@@ -0,0 +1,3 @@
+-- Rotface
+UPDATE `acore_world`.`creature_template` SET `flags_extra`=3221225473 WHERE  `entry`=36627;
```

Comment:
dont add flags like this. use the add operator. see https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits